### ISSUE TITLE
logbook support log http request in CURL format 

### ISSUE THREAD

**Issue by wyzssw** [zalando/logbook#160]
As a user, i want to print HttpClient request in CURL format which can easy to test by just execute at command line terminal, is this feature in your future plan？

**Comment by whiskeysierra** [comment]
> is this feature in your future plan

It now is. Any chance you have time to contribute this yourself?

**Comment by wyzssw** [comment]
Perhaps not in the coming half year，so sorry 😊

**Comment by whiskeysierra** [comment]
@wyzssw I created #166, please take a look.

**Comment by whiskeysierra** [comment]
@wyzssw I released 1.3.0, should be in central soon.

**Comment by wyzssw** [comment]
I tryed 1.3.0 and change format to `curl`
but the log print below
```java
2017-06-28 17:29:08.608 [http-nio-9235-exec-6] INFO  [spring-httplog:74] - curl -v -X GET 'http://10.10.11x.10x:9235/queryBu' -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8' -H 'accept-encoding: gzip, deflate' -H 'accept-language: zh-CN,zh;q=0.8,en;q=0.6,ja;q=0.4' -H 'cache-control: max-age=0' -H 'connection: keep-alive' -H 'host: 10.10.11x.10x:9235' -H 'upgrade-insecure-requests: 1' -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36'


2017-06-28 17:29:14.444 [http-nio-9235-exec-6] INFO  [spring-httplog:79] - Outgoing Response: f6c9971b-23cf-4555-ab30-3857f7450151
HTTP/1.1 200 OK
X-Application-Context: application:9235
Content-Length: 784
Date: Wed, 28 Jun 2017 09:29:14 GMT
Content-Type: application/json;charset=UTF-8
```
Which  `Coming Request: xxxxx` disappeared 

can you add  `Coming Request: xxxx` before curl log  ?
```java
2017-06-28 17:29:08.608 [http-nio-9235-exec-6] INFO  [spring-httplog:74] - Coming Request: f6c9971b-23cf-4555-ab30-3857f7450151
curl -v -X GET 'http://10.10.11x.10x:9235/queryBu' -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8' -H 'accept-encoding: gzip, deflate' -H 'accept-language: zh-CN,zh;q=0.8,en;q=0.6,ja;q=0.4' -H 'cache-control: max-age=0' -H 'connection: keep-alive' -H 'host: 10.10.11x.10x:9235' -H 'upgrade-insecure-requests: 1' -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Safari/537.36'


2017-06-28 17:29:14.444 [http-nio-9235-exec-6] INFO  [spring-httplog:79] - Outgoing Response: f6c9971b-23cf-4555-ab30-3857f7450151
HTTP/1.1 200 OK
X-Application-Context: application:9235
Content-Length: 784
Date: Wed, 28 Jun 2017 09:29:14 GMT
Content-Type: application/json;charset=UTF-8
```

**Comment by whiskeysierra** [comment]
The idea was to replace the request log with a curl statement.

**Comment by whiskeysierra** [comment]
I even thought about not logging the response at all in this case.

**Comment by whiskeysierra** [comment]
Ah, you're just missing the correlation id?

**Comment by wyzssw** [comment]
`Ah, you're just missing the correlation id?`
yes. i still want response-logging whenever curl or http format

**Comment by whiskeysierra** [comment]
Where would you expect it? Immediately before the curl command?

**Comment by wyzssw** [comment]
> Where would you expect it? Immediately before the curl command?

yes

**Comment by whiskeysierra** [comment]
@wyzssw I released 1.3.1 with the correlation id prepended.

**Comment by wyzssw** [comment]
Thanks
